content/chainguard/libraries/javascript/build-configuration.md

```diff
@@ -216,36 +216,155 @@ To change the packages, remove the `node_modules` directory and the
+The command results in the following line:
 
 ```
-yarn config set npmRegistryServer http://localhost:8081/repository/javascript-all/
-``` 
+npmRegistryServer: "https://repo.example.com:8443/repository/javascript-all/"
+```
+
+Check the [config set documentation](https://yarnpkg.com/cli/config/set) for
+more details such as authentication support.
+
+Example URLs:
+
+* JFrog Artifactory: https://example.jfrog.io/artifactory/javascript-all/
+* Sonatype Nexus: https://repo.example.com:8443/repository/javascript-all/
+* Direct access: https://libraries.cgr.dev/javascript/
+
+To change the packages, run the `yarn` command again. This forces an updated of
+all packages from the new registry and regeneration of the lock file.
+
+Now you can proceed with your development and testing. 
 
 <a id="yarn-classic"></a>
 
 ## Yarn Classic
 
-[Yarn Classic](https://classic.yarnpkg.com/) is the legacy release of Yarn using
-versions 1.x.
+[Yarn Classic](https://classic.yarnpkg.com/) is the legacy 1.x release of
+[Yarn](#yarn).
+
+Using yarn, dependencies to JavaScript packages are declared in a `package.json`
+file and separated into different scoped dependencies. development and runtime
+dependencies. The following block shows a minimal example with `react` and
+`node` as main runtime dependencies and `eslint` as development dependency:
+
+```json
+{
+  "name": "yarn-classic-example",
+  "version": "1.0.0",
+  "description": "A minimal example project for using yarn classic",
+  "main": "index.js",
+  "author": "Chainguard",
+  "license": "MIT",
+  "private": false,
+  "dependencies": {
+    "node": "^22.18.0",
+    "react": "^19.1.1"
+  },
+  "devDependencies": {
+    "eslint": "^9.36.0"
+  }
+}
+```
+
+By default, the packages are retrieved from the registry at
+`https://registry.yarnpkg.com` and stored locally in the `node_modules`
+directory after running `yarn`. This operation also creates the `yarn.lock`
+file.
+
+Note that dependency versions are typically declared with the `^` before the
+version string. This indicates higher, compatible versions, following the
+semantic versioning scheme of the package, are used automatically. For example,
+the declaration of version `^22.18.0` for `node`, actually results in the use of
+version `22.20.0` or even a higher version once available and `yarn` is run.  
+
+Any dependency or dependency version changes require another install and
+therefore an update to the lock file. The lock file also encodes the checksum
+values in the `integrity` field and the download URL in the `resolved` field for
+each module.
+
+To switch a project to use Chainguard Libraries for JavaScript, set the registry
+URL to point to your repository manager in your user `.yarnrc` file: 
+
+```shell
+yarn config set registry https://repo.example.com:8443/repository/javascript-all/
+```
+
+The command results in the following line:
+
+```
+registry "https://repo.example.com:8443/repository/javascript-all/"
+```
 
--->
+Check the [.yarnrc
```

Comment:
I'd say "Check out the" or "Refer to the" docs

"Check" the docs, to my reading, implies like check it once, or check it as one would for an update, like now and then, when the directive should be more "explore this resource as needed"